build.sh

```diff
@@ -304,10 +304,10 @@ if completeBuild || hasArg cuml || hasArg pydocs; then
     fi
 fi
 
-if hasarg cuml-cpu; then
+if hasArg cuml-cpu; then
     cd ${REPODIR}/python
-    python setup.py build_ext --inplace -- -DCMAKE_MESSAGE_LOG_LEVEL=${CMAKE_LOG_LEVEL} ${CUML_EXTRA_CMAKE_ARGS} -- -j${PARALLEL_LEVEL:-1}
+    python setup.py build_ext --inplace -- -DCMAKE_MESSAGE_LOG_LEVEL=${CMAKE_LOG_LEVEL} -DCUML_CPU=ON -- -j${PARALLEL_LEVEL:-1}
```

Comment:
Are we okay to drop `CUML_EXTRA_CMAKE_ARGS` entirely? I thought this was a pass-through allowing the specification of such args using the build.sh script.

Reply:
I don't think there's a usecase currentlufor any extra cmake args for the CPU build, but will add it back so we have it in case